moviebot/run.py

```diff
@@ -17,101 +28,47 @@
 _DEFAULT_CONFIG = "config/moviebot_config.yaml"
 
 
-def _validate_file(file_name, file_type):
-    """Checks if the file is valid and is present
-
-    Args:
-      file_name: name/path of the file provided
-      file_type: the type/extension of the file
-
-    Returns:
-      error and details
-
-    """
-    if isinstance(file_name, str):
-        if os.path.isfile(file_name):
-            details = file_name.split(".")
-            if len(details) > 1:
-                if details[-1] == file_type:
-                    return "NoError"
-                else:
-                    return "ValueErrorType:{}".format(details[-1])
-            else:
-                return "ValueErrorType:{}".format(details[-1])
-        else:
-            return "FileNotFoundError"
-    else:
-        return "ValueError"
-
-
-def arg_parse(args=None):
-    """Parses the arguments in the configuration file
+def parse_args(args=None) -> argparse.Namespace:
+    """Parse command line arguments.
 
     Args:
-      args: configuration file for the dialogue (Default value = None)
+        args (optional): List of arguments to parse. If not provided, uses
+            sys.argv[1:]. Defaults to None.
```

Comment:
sys.argv[1:] is not a param?

Reply:
ArgumentParser defaults to `sys.argv[1:]`.